readme.md

```diff
@@ -13,16 +13,25 @@
 
 ## Usage
 
-1. In settings, paste the contents of one of the theme files (e.g. `rose-pine.json`) in `schemes`
+1. In settings json, paste the contents of one of the scheme files (e.g. `rose-pine.scheme.json`) in `schemes` and the corresponding theme (e.g. `rose-pine.theme.json`) in `themes`
```

Comment:
Could we replace "In settings json" with the actual filename? E.g. In `settings.json`

Reply:
Thanks for reviewing this so quickly. Sure I can change that but what do you think of the following wording?

## Usage

1. Open `settings.json` from Windows Terminal
2. In the `schemes` section of the file, paste the contents of your chosen scheme file (e.g. `rose-pine.scheme.json`)
3. Navigate to the `themes` section and paste the contents of the corresponding theme file (e.g. `rose-pine.theme.json`).
4. Update `colorScheme` within the `profiles` section to include your chosen scheme:

```json
{
    "profiles": {
        "defaults": {
            "colorScheme": "rose-pine"
        }
    }
}
```

5. Update `theme` to use your chosen theme:

```json
{
    "theme": "rose-pine"
}
```

Reply:
I like that wording, easier to follow. Below is an example of indenting the code blocks to align with the steps but honestly either option is fine. Code blocks under list items usually change the spacing either way so if you prefer how you have it formatted then no change.

```
1. Step 1
2. Step 2

    ```js
    function add() {}
    ```

3. Step 3
```

1. Step 1
2. Step 2

    ```js
    function add() {}
    ```

3. Step 3

Reply:
Indenting looks nicer. I included that in the following commit.